modules/apps/commerce/commerce-product-service/service.xml

```diff
@@ -1311,6 +1311,34 @@
 			<finder-column name="groupId" />
 		</finder>
 	</entity>
+	<entity local-service="true" name="CPSpecificaitonOptionListTypeDefinitionRel" remote-service="true" table="CPSOListTypeDefinitionRel">
```

Comment:
There's a typo here:
CPSpecificaitonOptionListTypeDefinitionRelId => CPSpecificationOptionListTypeDefinitionRelId

Reply:
_aiton_ instead of _ation_